docker-compose.yaml

```diff
@@ -170,36 +177,43 @@ services:
       LND_SOCKET: ${LND_SOCKET}
       VIRTUAL_HOST: api.local
       TENDERMINT_URI: http://abci:26657
+      HOT_WALLET_ADDRESS: ${HOT_WALLET_ADDRESS}
+      LND_MACAROON: '12345'
```

Comment:
Should these be parameterized?

Reply:
They are in the normal container but this is just for testing which doesn't really need real connection details, but just wants to see that something is there.

Reply:
nvm, doesn't seem to break anything if it is parameterized and nothing is set in the env file. I'll change it.